src/darwin/Framework/CHIP/MTRDeviceControllerDelegateBridge.mm

```diff
@@ -117,6 +118,7 @@
 void MTRDeviceControllerDelegateBridge::OnCommissioningComplete(chip::NodeId nodeId, CHIP_ERROR error)
 {
     MTR_LOG_DEFAULT("DeviceControllerDelegate Commissioning complete. NodeId %llu Status %s", nodeId, chip::ErrorStr(error));
+    MATTER_LOG_METRIC_END(kMTRPairingSetup);
```

Comment:
same here - there is no metric here, just the implied "compute duration from the previous start". We should probably have some way to convey that and likely not related to "metric".

Reply:
End can always take result of the operation being tracked from the Begin. I will update to the error here.